### PR TITLE
Rogue update

### PR DESCRIPTION
**Pull request by Smufrik**
edit in assa rotation
added vendetta
fixed a coding miss in the lua that made hunger for blood and kick to be a comment

### CODE REVIEW

File: Cataclysm/Rogue.lua [hunk 1]
```diff
@@ -1092,7 +1098,8 @@ spec:RegisterAbilities( {
     },
 
 
-    --[[ Enrages you, increasing all damage caused by 5%.  Requires a bleed effect to be active on the target.  Lasts 1 min.
+    -- Enrages you, increasing all damage caused by 5%.  Requires a bleed effect to be active on the target.  Lasts 1 min.
```
Comment (Hekili) [inline Cataclysm/Rogue.lua]: Show me Hunger for Blood in-game?  Spell ID 63848 doesn't exist on Wowhead.

Comment (Smufrik) [reply Cataclysm/Rogue.lua]: Aaah then we might aswell take it away. Eatherway it was busting kick

File: Cataclysm/APLs/RogueCombat.simc [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-actions+=/sinister_strike,if=buff.deep_insight.down&buff.shallow_insight.down
-actions+=/sinister_strike,if=combo_points<5
-actions+=/actions.aoe,if=active_enemies>3
 
 
 actions.aoe+=/blade_flurry,if=active_enemies>1&!buff.blade_flurry.up
 actions.aoe+=/fan_of_knives,if=active_enemies>3&energy<50&buff.deep_insight.up
 actions.aoe+=/actions
-actions.aoe+=/cancel_buff,name=blade_flurry,if=active_enemies=1&buff.blade_flurry.up
+actions.aoe+=/cancel_buff,name=blade_flurry,if=active_enemies=1&buff.blade_flurry.upactions+=/sequence,name=opening,if=buff.slice_and_dice.down&combo_points=0,actions=sinister_strike,slice_and_dice
```
Comment (Hekili) [inline Cataclysm/APLs/RogueCombat.simc]: Sequences aren't supported adn there seems to be a missing line-break here.

Comment (Smufrik) [reply Cataclysm/APLs/RogueCombat.simc]: oh thats a typo by me

**Review by Hekili — APPROVED** [review]

**Pull request merged**